Tests/RockLib.Logging.Tests/DependencyInjection/LoggerBuilderTests.cs

```diff
@@ -124,6 +124,7 @@ public void AddContextProviderMethodSadPath()
         [InlineData(true), InlineData(false)]
         public void BuildMethodHappyPath1(bool customLogLevelResolver)
         {
+            var expectedLogLevel = customLogLevelResolver ? LogLevel.Error : LogLevel.Info;
```

Comment:
Not sure why you just don't pass that in with the data. I would be interested to hear from other @RockLib/maintainers what they think of this.

Reply:
It's just my personal opinion that it's better to keep parameters as dictations on the path the tests take vs. using them for expected result storage.

Admittedly, it's an evangelical approach to that opinion in a scenario like this. In any case, i'll change it to what you're requesting. It truly doesn't matter in the end - the fact that tests are there is what's important :).

Reply:
I also like the idea of using them for expected result storage.:). But as you said, it doesn't matter as long as the tests are there.  I'd be ok with either, but it might be better to stick with a pattern that we decide.